cobalt/browser/performance/public/mojom/performance.mojom

```diff
@@ -0,0 +1,21 @@
+// Copyright 2025 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+module performance.mojom;
+
+interface CobaltPerformance {
+  // Get the advertisingId for the device.
+  [Sync]
```

Comment:
Is there a reason for this interface to need to be Sync? I guess is due to WebApp limitations, but if so there's no mention of it in the PR description (which anyway now is basically empty).

Reply:
It needs to be sync because this API is used by player and they feel they need synchronous APIs